vms/platformvm/txs/executor/state_changes.go

```diff
@@ -207,29 +206,3 @@ func AdvanceTimeTo(
 	}
 	return changes, nil
 }
-
-func GetRewardsCalculator(
-	backend *Backend,
-	parentState state.Chain,
-	subnetID ids.ID,
-) (reward.Calculator, error) {
```

Comment:
By moving this into the interface we end up needing to implement it twice - this should already work for diffs and the base state... So unless this is for mocking the result of this function in tests I'm not sure this is cleaner